helm/alfresco-content-services/values.yaml

```diff
@@ -84,7 +84,9 @@ repository:
   command: []
   # -- Provide additional log statements by adding
   # classes and/or packages in a key:value fashion
-  extraLogStatements: {}
+  extraLogStatements:
```

Comment:
This was only for test purposes, I'm going to remove it during the first commit after the pull request review.